### PR TITLE
[NCL-1586] fixing TermBuildDriverRemoteTest and making it local 

### PR DESCRIPTION
**Pull request by michalszynkiewicz**

### CODE REVIEW

File: termd-build-driver/src/test/java/org/jboss/pnc/termdbuilddriver/TermdBuildDriverTest.java [hunk 1]
```diff
@@ -151,4 +157,45 @@ public void shouldSetLogsInBuildExecution() throws Exception {
 //        verify(buildExecutionMock, times(1)).clearLogsWebSocketLink();
     }
 
+    @Test(timeout = 60_000)
+    public void shouldFetchFromGitAndBuild() throws Exception {
+        //given
+        Path tmpRepo = Files.createTempDirectory("tmpRepo");
+        String repoPath = "file://" + tmpRepo.toAbsolutePath().toString() + "/test-repo";
+        ZipUtils.unzipToDir(tmpRepo, "/repo.zip");
```
Comment (slaskawi) [inline termd-build-driver/src/test/java/org/jboss/pnc/termdbuilddriver/TermdBuildDriverTest.java]: Looks good but I'm not a big fan for unzipping it here... Seems to be a bit fragile...

Would it be possible to create a directory in `src/test/resources` with almost empty `.git` and simulate that everything is there?

Comment (slaskawi) [inline termd-build-driver/src/test/java/org/jboss/pnc/termdbuilddriver/TermdBuildDriverTest.java]: Clarified - we can't use a synthetic git repo here.

**Comment by slaskawi** [comment]
:+1

File: termd-build-driver/src/test/java/org/jboss/pnc/termdbuilddriver/TermdBuildDriverTest.java (hunk 1 above)
Comment (matejonnet) [inline termd-build-driver/src/test/java/org/jboss/pnc/termdbuilddriver/TermdBuildDriverTest.java]: Do we need to test git at all? It is important that we test communication with BuildAgent. From a BuildDriver perspective it should be the same if you run `git clone` or `whoami`, important is that command is executed and result received.

**Pull request merged**